.gitpod.yml

```diff
@@ -1,3 +1,17 @@
 tasks:
   - init: npm install
-    command: npm run dev
\ No newline at end of file
+    command: npm run dev
+
+ports:
+  - port: 3000
+    onOpen: open-browser
+
+vscode:
+  extensions:
+    - svelte.svelte-vscode
+
+github:
+  prebuilds:
+    master: true
```

Comment:
Hey, this PR looks perfect! Thanks for this 🙂 

Can you explain this to me? Does master here stand for the default repo or does it have any other meaning?